cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -227,22 +230,26 @@ def reset_states():
 @when_not('kubernetes.dashboard.available')
 def launch_kubernetes_dashboard():
     ''' Launch the Kubernetes dashboard. If not enabled, attempt deletion '''
-    manifest = '/etc/kubernetes/addons/dashboard.yaml'
+    templates = [
+        'kubernetes-dashboard.yaml',
+        'influxdb-grafana-controller.yaml',
+        'influxdb-service.yaml',
+        'grafana-service.yaml',
+        'heapster-controller.yaml',
+        'heapster-service.yaml'
+    ]
     if hookenv.config('dashboard'):
```

Comment:
Perhaps this configuration name does not make sense any longer. Maybe we need to deprecate and change it to enable-addons or something like that.